test/test_helper.rb

```diff
@@ -92,19 +92,19 @@ def test_order_hour_of_day_order_reverse
     assert_equal 0, User.group_by_hour_of_day(:created_at).order("hour_of_day desc").reverse_order.count.keys.first
   end
 
-  def test_table_name
+  def test_group_by_day_count_is_empty_for_empty_records
```

Comment:
This test is to ensure there's not an error when using the table name as part of the column name.